examples/README.md

```diff
@@ -1,15 +1,20 @@
 # eBPF Examples
 
-- [kprobe](kprobe/) - Attach a program to the entry or exit of an arbitrary kernel symbol (function).
-- [kprobepin](kprobepin/) - Reuse a pinned map for the kprobe example. It assumes the BPF FS is mounted at `/sys/fs/bpf`.
-- [uretprobe](uretprobe/) - Like a kprobe, but for symbols in userspace binaries (e.g. `bash`).
-- [tracepoint](tracepoint/) - Attach a program to predetermined kernel tracepoints.
-- [ringbuffer](ringbuffer/) - Like the kprobe example, but using a RINGBUF map type.
-- Add your use case(s) here!
+* Kprobe - Attach a program to the entry or exit of an arbitrary kernel symbol (function).
+  * [kprobe](kprobe/) - Kprobe on `sys_execve` using bpf2go.
```

Comment:
I would omit mentioning the specific symbols like `sys_execve` here and in subsequent bullets, as they get changed here and there, and this will inevitably go stale. It also doesn't add much value to the description.

Otherwise LGTM!

Reply:
Changed